### DISCUSSION TITLE
Browser as device

### DISCUSSION THREAD

**Discussion by dCypherNx** [samuelthng/t-house-blueprints#36] [Q&A]
is there a way to use any open browser as device ?

**Comment by samuelthng** [comment]
This is currently unsupported by the notifications script as it focuses only on sending notifications to the HA Companion mobile app.

You can easily call the notification service using persistent notifications however. That would appear in your Home Assistant notifications tray.

Hope this helps.

**Comment by dCypherNx** [comment]
Unfortunately it doesn't help. I want to send a confirmation notification only to whoever calls the script, not to all users.
